tools/clang/tools/dxil2spv/lib/CMakeLists.txt

```diff
@@ -1,25 +1,26 @@
 # This file is distributed under the University of Illinois Open Source License. See LICENSE.TXT for details.
 
-set( LLVM_LINK_COMPONENTS
+set(LLVM_LINK_COMPONENTS
```

Comment:
How does `LLVM_LINK_COMPONENTS` work?
I guess somehow the llvm cmake build uses static/dynamic linking with the listed libraries.
You removed `LLVM_LINK_COMPONENTS` from tools/clang/tools/dxil2spv/CMakeLists.txt. I am curious how the listed libraries here linked with what target library.

Another concern is building it in Windows. Do we only target Linux build?

Reply:
It's just a helper variable that does the same `llvm_map_components_to_libnames` later on in CMake functions: https://github.com/microsoft/DirectXShaderCompiler/blob/54924ce5170a3bcb9f0a99c6249ce63e05e141ba/cmake/modules/AddLLVM.cmake#L476

Just a cleanup that follows the last PR where I introduced a clang library layer, since it's how the rest of the codebase's CMake files link LLVM components (both for windows and Linux builds).